Source/Core/InputCommon/ControlReference/ControlReference.cpp

```diff
@@ -48,6 +47,17 @@ ParseStatus ControlReference::GetParseStatus() const
   return m_parse_status;
 }
 
+std::string ControlReference::GetExpression() const
+{
+  return m_expression;
+}
+
+void ControlReference::SetExpression(std::string expr)
+{
+  m_expression = expr;
```

Comment:
Yes, I do sometimes use instruction count as a metric for evaluating assembly code, since I'm not well-versed in reading assembly code, which is why I asked (I know you don't mean it, but when you phrase it like it comes off as dickish).

In this case, the instruction count is higher while the number and names of calls made is the same. Can you take a look at https://godbolt.org/g/3nyJqO?